repo/packages/H/hello-world/0/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "description": "An example implementation of a stateful service using the DC/OS SDK",
+  "framework": true,
+  "maintainer": "support@mesosphere.io",
+  "minDcosReleaseVersion": "1.7",
+  "name": "hello-world",
+  "packagingVersion": "3.0",
+  "postInstallNotes": "hello-world post install notes go here.",
```

Comment:
add 'preInstallNotes' property and add the link to `dcos/examples` at the end of it.